tools/etcd-dump-db/README.md

```diff
@@ -38,7 +38,7 @@ meta
 ```
 
 
-#### hash [data dir or db file path]
+#### hash `[data dir or db file path]`
```

Comment:
this does not look good. 

/cc @crawford can you patch marker to ignore `[]` checking in the title?